src/core/ONSAS_init.m

```diff
@@ -29,6 +29,7 @@
 
 % =================================================================
 %md process boundary conds information and construct BCsData struct
+global Conec
```

Comment:
is this needed?

Reply:
removed in [eb890ee](https://github.com/ONSAS/ONSAS.m/pull/539/commits/eb890ee6649551252ba7eb64341964262e81dc31)